packages/models-library/tests/test_project_nodes.py

```diff
@@ -48,7 +48,7 @@ def test_create_minimal_node_with_new_data_type(
 
     node = Node(**old_node_data)
     assert (
-        node.thumbnail
+        str(node.thumbnail)
```

Comment:
`assert node.thumbnail is not None` before this line is correct. And remove the `str` cast